terraform/transform_provider_test.go

```diff
@@ -434,69 +434,6 @@ func TestProviderConfigTransformer_grandparentProviders(t *testing.T) {
 	}
 }
 
-// pass a specific provider into a module using it implicitly
-func TestProviderConfigTransformer_implicitModule(t *testing.T) {
```

Comment:
I presume these have been removed because it's caught during the earlier configload validate now?

Reply:
Yes, these fail while loading the config, so there's no transformation to test.